docs/can-guides/topics/forms.md

```diff
@@ -490,252 +492,252 @@ There are many useful techniques for working with related data. Before diving in
-		{{/if}}
-
-		{{#if(years)}}
-			<select value:bind="year">
-				{{^year}}
-					<option value=''>Select a Year</option>
-				{{/year}}
-				{{#each years}}
-					<option value:from="this">{{this}}</option>
-				{{/each}}
-			</select>
-		{{else}}
-			<select disabled><option>Years</option></select>
-		{{/if}}
-
-		<div>
-			{{#each(vehicles)}}
-				<h2>{{name}}</h2>
-				<img src:from="thumb" width="200px"/>
 			{{/each}}
-		</div>
+		</select>
+	{{else}}
+		<select disabled><option>Loading Models</option></select>
+	{{/if}}
+{{else}}
+	<select disabled><option>Models</option></select>
+{{/if}}
+
+{{#if(years)}}
+	<select value:bind="year">
+		{{^year}}
+			<option value=''>Select a Year</option>
+		{{/year}}
+		{{#each years}}
+			<option value:from="this">{{this}}</option>
+		{{/each}}
+	</select>
+{{else}}
+	<select disabled><option>Years</option></select>
+{{/if}}
+
+<div>
+	{{#each(vehicles)}}
+		<h2>{{name}}</h2>
+		<img src:from="thumb" width="200px"/>
+	{{/each}}
+</div>
 ```
 @highlight 2,only
 
-> Note: you could also use a [can-define.types.get getter] for this property. The getter will be called once to get the initial value and will only be called again if an observable it is using changes. Since no observables are being used, using a `get` for the `makes` property will behave the same as using `default`.
+> **Note:** you could also use a [can-define.types.get getter] for this property. The getter will be called once to get the initial value and will only be called again if an observable it is using changes. Since no observables are being used, using a `get` for the `makes` property will behave the same as using `default`.
 > Using `get` also means that you can use the shorthand [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get getter syntax] so this:
 >```js
 > makes: {
 >   get: function() {
->     ...
+>     // ...
 >   }
 > }
 >```
 > can be written like this:
 >```js
 > get makes() {
->   ...
+>   // ...
 > }
 >```
 
 #### Using promises
 
 The `makes` property is set to the result of calling [can-ajax], which returns a [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise promise].
 
-This promise is decorated by [can-reflect-promise] so that in the view you can easily use metadata related to the promise's [https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md#states state]:
+This promise is decorated by [can-reflect-promise] so that in the view you can easily use metadata related to the promise’s [https://github.com/domenic/promises-unwrapping/blob/master/docs/states-and-fates.md#states state]:
 
-* `state` - one of "pending", "resolved", or "rejected"
-* `isPending` - true if the promise is neither resolved nor rejected, false otherwise.
-* `isResolved` - true if the promise is resolved, false otherwise.
-* `isRejected` - true if the promise is rejected, false otherwise.
+* `state` — one of `pending`, `resolved`, or `rejected`
```

Comment:
Since these are strings, maybe it should be:
> `"pending"`, `"resolved"`, or `"rejected"`

Reply:
I don’t think anyone would be confused if those strings weren’t wrapped in quotation marks, but for the sake of being consistent with can-reflect-promise, I’ve reverted back to what you had.